### DISCUSSION TITLE
I can't find Markor on PlayStore April 2023

### DISCUSSION THREAD

**Discussion by marker10** [gsantner/markor#2021] [General] [answered]
I can't find Markor on PlayStore anymore. Why?

**Comment by gsantner** [comment, marked as answer]
Because I've disabled it. Please install from F-Droid or from the APK attached to GitHub - these have always been the main source of Markor.

**Reply by greg4b** [reply]
I do have a corporate phone which does not allow installing from APK or F-Droid. So i can't use Markor :-(

If you would continue to publish on Playstore I could install from there.

Thanks, Austria#
